src/components/Detail/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams, useNavigate } from 'react-router-dom';
+import '../../css/details.css';
+import { fetchDetail } from '../../redux/detailReducer';
+import { formToggle } from '../../redux/reservations/reservationReducer';
+
+const Detail = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const { carid } = useParams();
+  const { data } = useSelector((state) => state.detailState.detail);
+  useEffect(() => {
+    dispatch(fetchDetail({ id: carid }));
+  }, [dispatch]);
+
+  const hanldeBack = () => {
```

Comment:
- [ ] I believe you have misspelled `handle` here, notice that you wrote `hanlde`.